src/include/rocblas_float8.h

```diff
@@ -22,10 +22,12 @@
 
 #ifndef ROCBLAS_FLOAT8_H
 #define ROCBLAS_FLOAT8_H
-#include <cstdint>
```

Comment:
why cstdint won't work?

Reply:
we need uint_8 data type, which defined in the stdint, so we need to include this head file.